packages/next/export/index.js

```diff
@@ -60,7 +62,20 @@ export default async function (dir, options, configuration) {
   for (const page of pages) {
     // _document and _app are not real pages
     // _error is exported as 404.html later on
-    if (page === '/_document' || page === '/_app' || page === '/_error') {
+    const isApiRoute = page.match(API_ROUTE)
+
+    // Warn about API pages export
+    if (isApiRoute && !apiPage) {
+      apiPage = true
+      log(`  API pages are not supported in export`)
```

Comment:
This message should be flashier IMO, plus it's missing an `err.sh` link explaining it.

Unactionable messages are against our logging guidelines.

If I'm a user, what do I do next? Not use `next export`? Delete my API Routes? Customize my export path map?